src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -17,178 +16,188 @@ import RequestorOnfidoStep from './RequestorOnfidoStep';
 import Form from '../../components/Form';
 import ScreenWrapper from '../../components/ScreenWrapper';
 import StepPropTypes from './StepPropTypes';
+import useLocalize from '../../hooks/useLocalize';
+
+const {reimbursementAccount, reimbursementAccountDraft, onBackButtonPress, getDefaultStateForField} = StepPropTypes;
 
 const propTypes = {
-    ...StepPropTypes,
+    ...{reimbursementAccount, reimbursementAccountDraft, onBackButtonPress, getDefaultStateForField},
```

Comment:
I think you can just do
```
const propTypes = {
      ...StepPropTypes,
      [...]
      }
```

Reply:
`StepPropsTypes` holds required typings for `withLocalize` that we need to omit here, that's why I've created this messy syntax :D